deployment/docker/victorialogs/fluentbit/README.md

```diff
@@ -23,39 +28,8 @@ Querying the data
 * [vmui](https://docs.victoriametrics.com/victorialogs/querying/#vmui) - a web UI is accessible by `http://localhost:9428/select/vmui`
 * for querying the data via command-line please check [these docs](https://docs.victoriametrics.com/victorialogs/querying/#command-line)
 
+Here is an example of fluentbit configuration
 
-the example of fluentbit configuration(`fluent-bit.conf`)
-
-```shell
-[INPUT]
-    name              tail
-    path              /var/lib/docker/containers/**/*.log
-    multiline.parser  docker, cri
-    Parser docker
-    Docker_Mode  On
-
-[INPUT]
-    Name     syslog
-    Listen   0.0.0.0
-    Port     5140
-    Parser   syslog-rfc3164
-    Mode     tcp
-
-[SERVICE]
-    Flush        1
-    Parsers_File parsers.conf
-
-[Output]
-    Name http
-    Match *
-    host victorialogs
-    port 9428
-    compress gzip
-    uri /insert/jsonline?_stream_fields=stream&_msg_field=log&_time_field=date
-    format json_lines
-    json_date_format iso8601
-    header AccountID 0
-    header ProjectID 0
-```
+https://github.com/VictoriaMetrics/VictoriaMetrics/blob/752eb347fe4e0343bfb99798dcba5742807a0614/deployment/docker/victorialogs/fluentbit/elasticsearch/fluent-bit.conf#L1-L31
```

Comment:
Is it possible including the whole file contents without the need to specify the exact lines range at `#L1-L31`? This will simplify maintenance of these docs in the future when the number of lines in `fluent-bit.conf` changes.

Reply:
I've tried using branch names instead of commit hashes and avoid using line ranges, but it's not rendering in each of these cases. other workarounds:
- upload it to our docs site and render with Hugo
- keep a link to a config file without rendering it in markdown

Reply:
We can simply keep a link and not rely on github feature of embedding code.